components/PolygonsAndMarkers.tsx

```diff
@@ -63,6 +82,17 @@ class PolygonsAndMarkers extends Component<markersAndPolygonsProps, any> {
             title={marker.title}
             description={marker.description}
             pinColor={marker.pinColor}
+            onPress={() => {
+              location.setLatitude(marker.coordinate.latitude);
+              location.setLongitude(marker.coordinate.longitude);
+              campus.setLocation(location);
+              building.setDescription(marker.description);
```

Comment:
ok so this is a strange one. In theory the markers shouldn't be holding the building description. In fact, looking at all the information the markers hold, they should actually be building objects. The way I would see this happening is in the CampusMarkers.tsx, instead of creating those objects, we should create building objects instead (new Building(......)). But to do it this way would take some time for refactoring. We will most likely have to do this to have access to building objects in the search. I will create another user story called 'Format Buildings' to handle this. We can discuss this more if you have any questions, but this is ok for now

Reply:
Yes, that is what we discussed last time to make the markers file into building objects with markers as attribute. It would be better to load a file with object building indeed with all necessary info in them. I did it that way because I did not want to refactor code that is out of this ticket, next ticket we can do the refactoring. To keep in mind for next meeting!